pages/en/contrib/style-guide.md

```diff
@@ -449,6 +449,44 @@ async function getDodo() {
 }
 ```
 
+### Always await the last promise in a function
+
+If the last line of a function handles a promise, always await that promise, regardless whether the result will be returned or not. (This makes it clear whether we did or did not intend to return the value, and dismisses any idea that we are still linking up the promise chain in pre-await style.)
+
+**Good**:
+
+```js
+async function openCircus() {
+  await openCarPark();
+  await openDoors();
+}
+```
+
+**Good**:
+
+```js
+async function getAnimal() {
+  return await getElephant();
+}
+```
+
+**Bad**:
+
+```js
+async function openCircus() {
+  await openCarPark();
+  return openDoors();
```

Comment:
Should we mention this style is not friendly to `try...catch`. For example:
```
async function openCircus() {
  try {
    await openCarPark();
    return await openDoors();
  } catch(err) {
    // rejected promise from openDoors will be trapped here
    throw err;
  }
```

```
async function openCircus() {
  try {
    await openCarPark();
    return openDoors();
  } catch(err) {
    // rejected promise from openDoors will NOT be trapped here
    throw err;
  }
```

Reply:
I have added a note about that.